be/src/exec/data_sink.cpp

```diff
@@ -173,6 +174,22 @@ Status DataSink::create_data_sink(RuntimeState* state, const TDataSink& thrift_s
         *sink = std::move(mcast_data_stream_sink);
         break;
     }
+    case TDataSinkType::Split_DATA_STREAM_SINK: {
```

Comment:
better to use all uppercase letters for naming, to maintain consistency with other types.